service/resource/vaultcrt/create.go

```diff
@@ -96,10 +96,8 @@ func (r *Resource) ensureVaultRole(customObject certificatetpr.CustomObject) err
 			AllowSubdomains:  AllowSubDomains,
 			AltNames:         key.AltNames(customObject),
 			ID:               key.ClusterID(customObject),
-			Organizations: []string{
-				key.ClusterComponent(customObject),
-			},
-			TTL: key.RoleTTL(customObject),
+			Organizations:    key.Organizations(customObject),
```

Comment:
I like this ease, but as I said before, we also have to set the orgs above when checking if the role exists.

Reply:
Good catch haven't seen it :)